nixos/modules/virtualisation/digital-ocean-config.nix

```diff
@@ -0,0 +1,81 @@
+{ config, lib, pkgs, options, ... }:
+with lib;
+let
+  cfg = config.virtualisation.digitalOceanImage;
+in
+{
+  imports = [
+    ../profiles/qemu-guest.nix
+  ];
+
+
+  fileSystems."/" = {
+    device = "/dev/disk/by-label/nixos";
+    autoResize = true;
+    fsType = "ext4";
+  };
+
+  boot.growPartition = true;
+  boot.kernelParams = [ "console=ttyS0" "panic=1" "boot.panic_on_fail" ];
+  boot.initrd.kernelModules = [ "virtio_scsi" ];
+  boot.kernelModules = [ "virtio_pci" "virtio_net" ];
+
+  # password "nixos"
+  users.users.nixos = {
```

Comment:
On amazon image and Openstack NixOS images, the root account is used by default. If a user wants a `nixos` user, it is possible either to build a custom image or reconfigure a deployed image. So, would it be possible to remove this account?